src/StyleTransfer.ts

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { Image } from 'react-native';
+import { StyleTransfer } from './native/RnExecutorchModules';
+import { ETError, getError } from './Error';
+
+interface Props {
+  modulePath: string | number;
+}
+
+interface StyleTransferModule {
+  error: string | null;
+  isModelLoading: boolean;
+  isModelRunning: boolean;
+  forward: (input: string) => Promise<string>;
+}
+
+export const useStyleTransfer = ({
+  modulePath,
+}: Props): StyleTransferModule => {
+  const [error, setError] = useState<null | string>(null);
+  const [isModelLoading, setIsModelLoading] = useState(true);
+  const [isModelRunning, setIsModelRunning] = useState(false);
```

Comment:
`isModelGenerating` would be better i think. Analogously to what we did with LLaMA. This change would also need to be done in ETModule.ts

Reply:
hmm, not so sure about that, this implies we would do also `isModelSegmenting` and `isModelDetecting` and so on, unless you suggest we should use `isModelGenerating` in all cases

Reply:
I had same thoughts as @mkopcins but we can stick to `isModelGenerating` as it should be understandable for everyone